warehouse/models/mart/gtfs/fct_vehicle_positions_trip_summaries.sql

```diff
@@ -1,55 +1,112 @@
 {{
     config(
-        materialized='incremental',
-        incremental_strategy='insert_overwrite',
-        partition_by={
-            'field': 'dt',
-            'data_type': 'date',
-            'granularity': 'day',
-        },
+        materialized='table',
         cluster_by='base64_url',
     )
 }}
 
-WITH vehicle_positions AS (
-    SELECT * FROM {{ ref('fct_vehicle_positions_messages') }}
-    WHERE {{ incremental_where(default_start_var='PROD_GTFS_RT_START') }}
+WITH vehicle_positions AS ( --noqa: ST03
+    SELECT * FROM {{ ref('int_gtfs_rt__vehicle_positions_trip_day_map_grouping') }}
+),
+
+vehicle_positions_no_lat_long AS ( --noqa: ST03
+    SELECT * EXCEPT (first_position_latitude, first_position_longitude, last_position_latitude, last_position_longitude)
+    FROM vehicle_positions
+),
+
+base_fct AS (
+    {{ gtfs_rt_trip_summaries(input_table = 'vehicle_positions_no_lat_long',
+    urls_to_drop = '("aHR0cDovL3d3dy5teWJ1c2luZm8uY29tL2d0ZnNydC92ZWhpY2xlcw==")',
+    extra_timestamp = 'vehicle')
+    }}
+),
+
+lat_long AS (
+    -- can also qualify but the point is that you will get two rows for trips that cross UTC date boundary
+    SELECT DISTINCT
+        key,
+        FIRST_VALUE(first_position_latitude)
+            OVER
+            (PARTITION BY key
+            ORDER BY COALESCE(min_vehicle_timestamp, min_header_timestamp, min_extract_ts)
+            ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING
```

Comment:
if you wanted, you could define a named window to use in these repetitive situations https://cloud.google.com/bigquery/docs/reference/standard-sql/window-function-calls#def_use_named_window

Reply:
https://github.com/cal-itp/data-infra/pull/2489/commits/df60317d8f357d197749897d0313776d27c888d6